src/celeritas/ext/GeantImporter.cc

```diff
@@ -239,33 +241,75 @@ import_particles(GeantImporter::DataSelection::Flags particle_flags)
 
 //---------------------------------------------------------------------------//
 /*!
- * Return a populated \c ImportElement vector.
+ * Return a populated pair of \c ImportElement and \c ImportIsotope vectors.
  */
-std::vector<ImportElement> import_elements()
+auto import_elements()
+    -> std::pair<std::vector<ImportElement>, std::vector<ImportIsotope>>
```

Comment:
Make a separate function `import_isotopes` that imports from `G4Isotope::GetIsotopeTable`. Otherwise you're potentially duplicating the same isotope.